nodejs12.x/cookiecutter-typescript-app-template/README.md

```diff
@@ -0,0 +1,50 @@
+# {{ cookiecutter.project_name }}
+
+This template generates a sample NodeJS 12.x project configured with TypeScript using the [Serverless Application Model (SAM)](https://github.com/awslabs/serverless-application-model).
+
+This template includes a full suite of tests using [MochaJs](https://mochajs.org/), [ChaiJs](https://www.chaijs.com/) and mocking of services using [Moq.ts](https://github.com/dvabuzyarov/moq.ts). The project is configured to lint the TypeScript using [eslint](https://eslint.org/).
+
+## Requirements
+
+* [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli)
+* 
+## ## Usage
+
+Generate a boilerplate tempalte in your current project directory using the following syntax:
+
+```
+$ sam init --runtime nodejs12.x
+```
+
+> **NOTE**: ``--name`` allows you to specify a different project folder name (`sam-app` is the default)
+
+Once the template is generated and you have installed the dependencies via `$ npm install` you can run the unit tests, lint the project and compile the project with the following NPM scripts.
+
+```
+$ npm run compile
+$ npm run test
+$ npm run lint
+```
```

Comment:
As I mentioned in the general conversation, it would be better if we can put these commands into a `Makefile`. We may still mention it here, but it will be easier for others to run build for this template.